### DISCUSSION TITLE
New event "integration ready"

### DISCUSSION THREAD

**Discussion by chemelli74** [home-assistant/architecture#754] [General]
**Problem**
During startup some integrations may be sometime slow to initialize, making impossibile to trigger an automation.

**Proposal**
Add a new even 'integration ready' to cove above scenario.

**Alternative**
None found :-(

**Comment by bdraco** [comment]
I'd love to have this since it would allow us to know when a config entry has finished reloading.

I think we would need a way to track all the tasks by which integration created them so we could tell when they are done.

Unfortunately its a bit difficult since there can be multiple instances of an integration.

- Created by a single config entry -- probably doable
- Created by a yaml platform - there really isn't a unique identifier to index by
